src/document/DocumentAnnotator.scss

```diff
@@ -11,7 +11,15 @@
         pointer-events: none;
     }
 
-    &.ba-is-highlighting {
+    .textLayer {
```

Comment:
I wonder if we should apply these styles along with some "discoverability" class style that is applied when the FF is enabled?

Reply:
That sounds good. 
Could you describe how this could be achieved?

I understand that we would be applying this block only if the feature is enabled. 
```
.textLayer {
        pointer-events: none; // Ensures that click events are received on the create region layer

        span {
            pointer-events: auto; // Text layer still receives text highlight events
        }
    }
```

Reply:
You can dynamically add `ba-discoverability-active` class to `textLayer` element. And change the scss to `textLayer.ba-discoverability-active { ... }`.